### PR TITLE
Test proc server control

### PR DESCRIPTION
**Pull request by John-Holt-Tessella**
### Description of work

Tests for ticket

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/4116

### Acceptance criteria

Test check changes.

### Unit tests

This is tests

### System tests

With IOC running and stopped check:

1. start ioc
1. stop ioc
1. restart ioc

### Documentation

None

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

File: test_start_stop_ioc_through_proc_control.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import os
+import unittest
+from utilities.utilities import g
+from six.moves import range
+import requests
+import time
+
+
+class TestPlotting(unittest.TestCase):
```
Comment (Tom-Willemsen) [inline test_start_stop_ioc_through_proc_control.py]: These tests appear to be totally unrelated to the ticket. Instead, they are an exact copy-paste of the tests in `test_plotting.py`.

Can you recheck your git commits?

**Comment by DominicOram** [comment]
I've moved the utils into the utilities file as if you don't import genie in the same way as the utilities file does it's harder to run the tests on the command line. Tests fail under master and pass with procserv changes so @John-Holt-Tessella if you're happy with my refactor I'll merge.

**Comment by John-Holt-Tessella** [comment]
@DominicOram I am happy with this, that is where I should have put them first time round. In general if you take up the ticket please do what you think is best.
